package.json

```diff
@@ -58,7 +62,21 @@
   "babel": {
     "presets": [
       "@babel/preset-env"
-    ]
+    ],
+    "env": {
+      "test": {
+        "presets": [
+          [
+            "@babel/preset-env",
+            {
+              "targets": {
```

Comment:
any reason, we are setting the targets here?

Reply:
Without setting a target of node, when running jest tests you get the following error:

>ReferenceError: regeneratorRuntime is not defined

This solution was provided in the following issue:
https://github.com/facebook/jest/issues/3126#issuecomment-483320742

Reply:
I'm tyring to figure out why we did not get that in webexComponent? maybe `pre-react` would resolve the issue?

Reply:
I tried everything including installing "@babel/preset-react". Not sure if storybook does something "magical" for jest tests or not.

Reply:
If you read the [following comment](https://github.com/facebook/jest/issues/3126#issuecomment-519845717) after the fix you suggested, it says that it will fix the test runner but it won't properly transpile the bundle.

I ran into this issue before, and the solution seemed to be to add polyfills for babel (`core-js` and `runtime-generator`). See [@babel/polyfill](https://babeljs.io/docs/en/next/babel-polyfill.html) docs